### PR TITLE
Fix: print token with newline

### PR DESCRIPTION
**Pull request by ESWZY**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

For a more friendly user interface when get token.

**Which issue(s) this PR fixes**:

Nope.

**Special notes for your reviewer**:

I just noticed when `keadm gettoken`, the next command line prompt begins with the end of token (which is difficult to get the token).

before

And I added a newline here:

after

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/cloud/gettoken.go [hunk 1]
```diff
@@ -74,7 +75,8 @@ func queryToken(namespace string, name string, kubeConfigPath string) ([]byte, e
 
 // showToken prints the token
 func showToken(data []byte, out io.Writer) error {
-	_, err := out.Write(data)
+	token := append(data, byte('\n'))
+	_, err := out.Write(token)
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/cloud/gettoken.go:79]: thanks for the fixing! @ESWZY 
how about `_, err := fmt.Fprintln(out, data)`?

Comment (ESWZY) [reply keadm/cmd/keadm/app/cmd/cloud/gettoken.go:79]: Sounds great!

**Comment by ESWZY** [comment]
@fisherxu Same effect as above.

**Pull request merged**